Makefile

```diff
@@ -20,14 +20,35 @@ release-shell: docker-running
 		--volume=$(CURDIR)/release:/usr/local/bin \
 		--entrypoint=/bin/bash cockpit/infra-release
 
+TAG := cockpit/infra-release:$(shell date --iso-8601)
 release-container: docker-running
 	docker build -t cockpit/infra-release:staged release
 	docker rm -f cockpit-release-stage || true
 	docker run --privileged --name=cockpit-release-stage \
 		--entrypoint=/usr/local/bin/Dockerfile.sh cockpit/infra-release:staged
 	docker commit --change='ENTRYPOINT ["/usr/local/bin/release-runner"]' \
-		cockpit-release-stage cockpit/infra-release
+		cockpit-release-stage $(TAG)
+	docker tag $(TAG) cockpit/infra-release:latest
 	docker rm -f cockpit-release-stage
+	@true
+
+release-push: docker-running
+	{ \
+	ID=`docker images -q cockpit/infra-release:latest`; \
+	if [ `echo "$$ID" | wc -w` -ne "1" ]; then \
+		echo "Expected exactly one image matching 'cockpit/infra-release:latest'"; \
+		exit 1; \
+	fi; \
+	TAGS=`docker images --format "table {{.Tag}}\t{{.ID}}" | grep $$ID | awk '{print $$1}'`; \
```

Comment:
this might seem a bit awkward, but I wanted to make sure that we get a known output format when scraping the image info